source/references/release-notes.txt

```diff
@@ -21,13 +21,47 @@ Release Notes
 Learn about new features, improvements, and fixes introduced in the
 following versions of the {+php-library+}:
 
+* :ref:`Version 2.2 <php-lib-version-2.2>`
+* :ref:`Version 2.1 <php-lib-version-2.1>`
 * :ref:`Version 2.0 <php-lib-version-2.0>`
 * :ref:`Version 1.21 <php-lib-version-1.21>`
 * :ref:`Version 1.20 <php-lib-version-1.20>`
 * :ref:`Version 1.19 <php-lib-version-1.19>`
 * :ref:`Version 1.18 <php-lib-version-1.18>`
 * :ref:`Version 1.17 <php-lib-version-1.17>`
 
+.. _php-lib-version-2.2:
+
+What's New in 2.2
+-----------------
+
+The {+library-short+} v2.2 release includes the following features,
+improvements, and fixes:
+
+- Adds the :phpmethod:`MongoDB\Model\CollectionInfo::isView()` method to
+  determine whether a collection is a :manual:`view </core/views/>`.
+
+.. _php-lib-version-2.1:
+
+What's New in 2.1
+-----------------
+
+.. important:: Breaking Changes
+
+   The {+library-short+} v2.1 release introduces the following breaking
+   changes:
+
+   - Drops support for {+mdb-server+} 4.0. The minimum supported
+     {+mdb-server+} version is 4.2.
+
+The {+library-short+} v2.1 release includes the following features,
+improvements, and fixes:
+
+- Adds a *client* bulk write API to perform write
+  operations on multiple databases and collections in the same call. To learn
+  more about this feature, see the :ref:`php-client-bulk-write`
```

Comment:
this link will work once comp-cov is merged into master so you can leave it as is